### ISSUE TITLE
Document need to quote globs

### ISSUE THREAD

**Issue by makannew** [stylelint/stylelint#6395]
### What steps are needed to reproduce the bug?

1. Create 3 css files with some invalid code and put them in nested folders.
  styles > fileA, folder1
  styles > folder1 > fileB, folder2
  styles > folder1 > folder2 > fileC                                                           

3. run ```npm run lint```  on windows machine
4. run same command on Mac os
5. On Mac we see one error caught by stylelint against 3 error on Windows machine

### What Stylelint configuration is needed to reproduce the bug?

This is about CLI command issue so I used a basic config just for demonstration.

"package.json" file:
```
{
  "name": "stylelint-bug",
  "version": "1.0.0",
  "description": "Reproducing inconsistency between Mac and Windows",
  "main": "index.js",
  "scripts": {
    "lint": "stylelint styles/**/*.css"
  },
  "author": "Makan Edrisi",
  "license": "ISC",
  "devDependencies": {
    "stylelint": "^14.13.0",
    "stylelint-config-standard": "^28.0.0"
  }
}

```

".stylelintrc.json" file:
```
{
  "extends": "stylelint-config-standard"
}
```

### How did you run Stylelint?

npm run lint

### Which version of Stylelint are you using?

14.13.0

### What did you expect to happen?

It should report same result on Windows machine as Mac machine

### What actually happened?

It seems style lint used glob pattern on Mac machine, so it only looked at styles and checked only one file.
But on windows machine it recursively checked all those folders and checked all 3 files.

I also found https://github.com/stylelint/stylelint/issues/4609 which seems happened because of same reason.

### Proposal to fix the bug

Changing the script to ```"lint": "stylelint styles/**/**/**/*.css"```solves this problem.
But depends on how many nested folder we have we should add more "/**/" to glob pattern works on Mac as well.

As [this article](https://medium.com/@jakubsynowiec/you-should-always-quote-your-globs-in-npm-scripts-621887a2a784)  suggests I propose the stylelint cli internally use [node-glob](https://github.com/isaacs/node-glob) to be consistent across different os.

**Comment by makannew** [comment]
@jeddy3 Thanks for quick reply, that's helpful :)
Also It would be nice if stylelint CLI can handle it internally or at least throw an error. It took several hours to me as I didn't have any clue why stylelint has different result on Mac os vs windows.
